HW8/MariiaDiachuk/08_Practical_Task_Structure_Project/models/__init__.py

```
@@ -0,0 +1,4 @@
+from .admin import *
```

Ruff (0.8.2)

`from .admin import *` used; unable to detect undefined names

(F403)